homeassistant/components/xiaomi_aqara/sensor.py

```diff
@@ -112,3 +112,8 @@ def parse_data(self, data, raw_data):
         else:
             self._state = round(value, 1)
         return True
+
+    def update(self):
```

Comment:
With this change the `update_entity` service will have an effect and users that are affected by the bad firmware can schedule their own update interval with automations. We then  don't need to set polling to true for all xiaomi aqara entities.